test/index.js

```diff
@@ -105,24 +116,34 @@ describe('envvar.boolean', function() {
 
 describe('envvar.number', function() {
 
-  it('coerces value to number', function() {
-    process.env.FOO = '42';
-    eq(envvar.number('FOO'), 42);
+  it('throws if applied to too few arguments', function() {
```

Comment:
I'd like to continue to support 0.10, since the cost is so low for a project we touch so infrequently.